xlators/cluster/dht/src/dht-lock.c

```diff
@@ -191,6 +191,8 @@ dht_lock_frame(call_frame_t *parent_frame)
     if (lock_frame)
         set_lk_owner_from_ptr(&lock_frame->root->lk_owner, parent_frame->root);
 
+    /* Set negative pid to conisder lock fop as an internal fop */
+    lock_frame->root->pid = GF_CLIENT_PID_NO_ROOT_SQUASH;
```

Comment:
shouldn't this be under the 'if' clause in line 191?

Reply:
Yes, i will update in next patch.